modules/solid_mechanics/src/timeintegrators/ExplicitMixedOrder.C

```diff
@@ -162,6 +157,18 @@ ExplicitMixedOrder::solve()
   _nonlinear_implicit_system->nonlinear_solver->converged = converged;
 }
 
+void
+ExplicitMixedOrder::evaluateRHSResidual()
+{
+  // Compute the residual
+  _explicit_residual->zero();
```

Comment:
I don't even think we need this zero? I think we talked about this that computeResidual zeros it out anyway?